docs/usage/getting-started/running.md

```diff
@@ -81,9 +83,10 @@ The Renovate team provide a ["Renovate Runner"](https://gitlab.com/renovate-bot/
 This supports both `gitlab.com` and self-hosted GitLab.
 Details for how it works can be found in the project.
 
-#### Mend Renovate On-Premises
+#### Mend Renovate Community Edition / Enterprise Edition
 
-Mend Renovate On-Premises (WSOP) started out as a commercial product "Renovate Pro", but was renamed and made free to use when Renovate became a part of Mend (formerly WhiteSource) in 2019.
+Mend Renovate Community Edition (Renovate CE) and Enterprise Edition (Renovate EE) are commercial offerings of Renovate for self-hosted users.
```

Comment:
```suggestion
Mend Renovate Community Edition (Renovate CE) and Enterprise Edition (Renovate EE) are non-Open Source offerings of Renovate for self-hosted users.
```

I'm not sure the best way to phrase this, but the term "commercial" usually implies that it costs money, while CE does not